[0.x] Documentation updates (#35)
laravel/vite-plugin: UPGRADE.md

        //     },
        // }),
    ],
});
```

If you are building an SPA, you will get a better developer experience by removing the CSS entry point above and [importing your CSS from Javascript](#importing-your-css-from-your-javascript-entry-points).

#### Update Aliases

> If you are migrating aliases from your Mix config to Vite, you should ensure that the paths start with a `/`, e.g. `resources/js` would become `/resources/js`:

Author: ```suggestion
If you are migrating aliases from your `webpack.mix.js` file to your `vite.config.js` file, you should ensure that the paths start with `/`. For example, `resources/js` would become `/resources/js`:
```

```js
export default defineConfig({
    plugins: [
        laravel([
            'resources/css/app.css',
            'resources/js/app.js',
        ]),
    ],
    resolve: {
        alias: {
            '@': '/resources/js'
        }
    }
});
```

> For convenience, the Laravel Plugin automatically injects an `"@"` alias that points to the `/resources/js` directory, so if this is the only alias you specify, you can omit it from your `vite.config.js`.

Author: ```suggestion
For your convenience, the Laravel Plugin automatically adds an `@` alias for your `/resources/js` directory. If you do not need to customize your aliases, you may omit this section from your `vite.config.js` file.
```
